From 148fc05557975267b05ebe670822793999ea3e0a Mon Sep 17 00:00:00 2001
From: Craig Bradney <mrb@scribus.info>
Date: Mon, 5 Jun 2023 21:15:35 +0000
Subject: [PATCH 11/13] Fix typo

git-svn-id: svn://scribus.net/trunk/Scribus@25504 11d20701-8431-0410-a711-e3c959e3b870
(cherry picked from commit bd2ba29c16218bc715cde93109e6c0b9be48e4be)
---
 CMakeLists_Dependencies.cmake | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/CMakeLists_Dependencies.cmake
+++ b/CMakeLists_Dependencies.cmake
@@ -8,7 +8,7 @@ if (WITH_PODOFO)
 	if (LIBPODOFO_FOUND)
 		message("PoDoFo found OK")
 		set(HAVE_PODOFO ON)
-		if(LIBPODOFO_VERSION VERSION_GREATER "0.10.0")
+		if(LIBPODOFO_VERSION VERSION_GREATER_EQUAL "0.10.0")
 			message(STATUS "PoDoFo Version:" ${LIBPODOFO_VERSION})
 			if (CMAKE_CXX_STANDARD LESS 17)
 				message(STATUS "C++17 is the minimum C++ standard since podofo 0.10.0")
